examples/utils/migration-tools/src/simpleLoader/README.md

```diff
@@ -0,0 +1,24 @@
+# `SimpleLoader`
+
+This package provides a `SimpleLoader` class, which wraps a `Loader` with a simpler interface.  This simpler interface is used by the `Migrator` during migration.
+
+```ts
+// Creating the SimpleLoader using Tinylicious
+const loader = new SimpleLoader({
+	urlResolver: new InsecureTinyliciousUrlResolver(),
+	documentServiceFactory: new RouterliciousDocumentServiceFactory(
+		new InsecureTinyliciousTokenProvider(),
+	),
+	codeLoader: new DemoCodeLoader(),
+	generateCreateNewRequest: createTinyliciousCreateNewRequest,
+});
+
+// Creating and attaching a new container
+const { container, attach } = await loader.createDetached("one");
+id = await attach();
+
+// Loading an existing container
+const container = await loader.loadExisting(id);
+```
+
+TODO: Can the `Migrator` take a normal `Loader` and wrap it itself to avoid teaching a new concept here?
```

Comment:
My two cents - I'm in favor of using regular `Loader`. If my understanding is correct, we are basically already wrapping `Loader` inside of `Migrator` and it would overall be simpler to avoid another wrapper. 

I could be wrong on the motivation behind `SimpleLoader`, but I also think there can be downsides from wrapping `Loader`, since IMO using `Loader` directly gives us more motivation to improve its flaws/usability.

Reply:
I think I agree - I got to SimpleLoader just by removing the parts that were duplicative with the concept of entryPoint and this is what was left.  But at this point it's mostly just a friendly API wrapper rather than any real functionality.  I'll open a separate work item to consider dropping all the way to plain Loader.

Reply:
AB#25113 for that consideration